spectacles/validators/sql.py

```diff
@@ -1,17 +1,37 @@
 from dataclasses import dataclass
 from tabulate import tabulate
 from typing import Union, Dict, Any, List, Optional
+import itertools
 import time
+from spectacles.utils import chunks
 from spectacles.client import LookerClient
 from spectacles.lookml import Dimension, Explore, Project
 from spectacles.exceptions import SpectaclesException, SqlError
 from spectacles.logger import GLOBAL_LOGGER as logger
 from spectacles.printer import print_header
 
+CHUNK_SIZE = 10
```

Comment:
Should we make this dynamic? This feels like overkill for a really small explore, and maybe not enough for a large one.

Reply:
Oh. I've actually misunderstood this. We run in chunks of 10, not 10 chunks. Right?

Reply:
Chunks of 10, correct. I'm not at all attached to this number. It should probably be way larger. I'm not sure if we need to make it configurable but we probably should tune it.